staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion_test.go

```diff
@@ -530,14 +549,22 @@ var conversionMultiVersionFixture = &apiextensionsv1beta1.CustomResourceDefiniti
 		Scope: apiextensionsv1beta1.NamespaceScoped,
 		Versions: []apiextensionsv1beta1.CustomResourceDefinitionVersion{
 			{
+				// storage version, same schema as v1alpha1
```

Comment:
@jpbetz note this additional version. It looks unnecessary here, but becomes necessary as soon as we enable validation and pruning. Then the schemas matter and we cannot do noop conversion from v1beta1 to v1beta2, only between v1alpha1 and v1beta1. I pulled this change from https://github.com/kubernetes/kubernetes/pull/77922 here to avoid rewriting all of this again.